templates/visuals/default/src/visual.ts

```diff
@@ -39,5 +39,9 @@ module powerbi.extensibility.visual {
             console.log('Visual update', options);
             this.target.innerHTML = `<p>Update count: <em>${(this.updateCount++)}</em></p>`;
         }
+
+        public enumerateObjectInstances(options: EnumerateVisualObjectInstancesOptions): VisualObjectInstance[] | VisualObjectInstanceEnumerationObject {
+            return VisualSettings.enumerateObjectInstances(VisualSettings.getDefault(), options);
```

Comment:
You shouldn't use ```VisualSettings.getDefault()```.
You have to do the following:
* Use ```VisualSettings.parse```
* Save result of the parsing to a variable of the class
* Use the variable instead  ```getDefault```